### PR TITLE
minor(helm readme): use star instead of the exact version 

### PR DESCRIPTION
**Pull request by rokroskar**
Originally submitted by @jkremser as #578.

### CODE REVIEW

File: charts/README.rst [hunk 1]
```diff
@@ -4,6 +4,26 @@ Helm Charts for Deploying RENKU on Kubernetes
...
+
+Deploying from a Helm repository
+--------------------------------
+
+.. code-block:: console
+
+    $ minikube start
+    $ helm init
+    $ helm repo add renku https://swissdatasciencecenter.github.io/helm-charts/
+    $ helm upgrade --install renku --namespace renku \
```
Comment (pameladelgado) [inline charts/README.rst]: I think this command might need more arguments... 
What we currently have in our [charts README](https://github.com/SwissDataScienceCenter/renku/tree/master/charts#testing-locally) is :
```helm upgrade --install \
    renku ./renku \
    --namespace renku \
    -f minikube-values.yaml \
    --set global.renku.domain=$(minikube ip) \
    --set ui.gitlabUrl=http://$(minikube ip)/gitlab \
    --set ui.jupyterhubUrl=http://$(minikube ip)/jupyterhub \
    --set ui.gatewayUrl=http://$(minikube ip)/api \
    --set gateway.keycloakUrl=http://$(minikube ip) \
    --set gateway.gitlabUrl=http://$(minikube ip)/gitlab \
    --set notebooks.jupyterhub.hub.extraEnv[0].value=http://$(minikube ip)/gitlab \
    --set notebooks.jupyterhub.hub.services.gateway.oauth_redirect_uri=http://$(minikube ip)/api/auth/jupyterhub/token \
    --set notebooks.jupyterhub.auth.gitlab.callbackUrl=http://$(minikube ip)/jupyterhub/hub/oauth_callback \
    --set notebooks.gitlab.registry.host=10.100.123.45:8105 \
    --set gitlab.registry.externalUrl=http://10.100.123.45:8105/ \
    --set graph.gitlab.url=http://$(minikube ip)/gitlab \
    --timeout 1800```

Comment (rokroskar) [reply charts/README.rst]: it's still in this readme, just lower down.

Comment (rokroskar) [reply charts/README.rst]: but you're right, we should make this top example very generic, not minikube-specific.

File: charts/README.rst [hunk 2]
```diff
@@ -4,6 +4,26 @@ Helm Charts for Deploying RENKU on Kubernetes
...
 
+
+Deploying from a Helm repository
+--------------------------------
+
+Create a values file using ``renku/values.yaml`` as a template. Then run:
+
+.. code-block:: console
+
+    $ minikube start
```
Comment (pameladelgado) [inline charts/README.rst]: maybe lose this line as well?
```suggestion
```

File: charts/README.rst (hunk 1 above)
Comment (pameladelgado) [reply charts/README.rst]: great, it's cleaner this way! ✨

**Pull request merged**